templates/docs/examples/templates/z-index.html

```diff
@@ -4,7 +4,7 @@
     <div class="p-navigation__banner">
       <div class="p-navigation__logo">
         <a class="p-navigation__item" href="#">
-          <img class="p-navigation__image" src="https://assets.ubuntu.com/v1/20673d9f-lxd_primary--for-dark-bg.svg" alt="" width="95" />
```

Comment:
This is a place where adding alt makes sense, because there is no other text representation of each link.

But I'd say the alt should say "LXD", "Kubernetes", etc (without the 'icon') - same as a label on the link would.